### PR TITLE
Doc: virtio-blk: add description of boot device option

### PR DESCRIPTION
**Pull request by binbinwu1**
Add the description of option 'b' to indicate the boot device including
the bootable image.

Tracked-On: #1917
Signed-off-by: Binbin Wu <binbin.wu@intel.com>

### CODE REVIEW

File: doc/developer-guides/hld/virtio-blk.rst [hunk 1]
```diff
@@ -65,6 +65,9 @@ The device model configuration command syntax for virtio-blk is::
 
    -s <slot>,virtio-blk,<filepath>[,options]
 
+- ``b``: needed when use vsbl as the virtual bootloader, to indicate the
```
Comment (dbkinder) [inline doc/developer-guides/hld/virtio-blk.rst]: Can you provide an example use of this ``b`` parameter?  Where do you put the ``b`` (and its parameters) in the syntax description above?

Also, edit the wording to be something like:
```
- ``b``: when using ``vsbl`` as the virtual bootloader, this indicates the
  bootable device and the bootable image location. If used, it must be
  the first option.
```

Comment (binbinwu1) [reply doc/developer-guides/hld/virtio-blk.rst]: updated, please help to have a check.

File: doc/developer-guides/hld/virtio-blk.rst [hunk 2]
```diff
@@ -63,8 +63,11 @@ Usage:
 
 The device model configuration command syntax for virtio-blk is::
 
-   -s <slot>,virtio-blk,<filepath>[,options]
+   -s <slot>,virtio-blk,[,b,]<filepath>[,options]
 
+- ``b``: needed when use ``vsbl`` as the virtual bootloader, to indicate the
```
Comment (dbkinder) [inline doc/developer-guides/hld/virtio-blk.rst]: ah OK.  This wording would be better:
```
- ``b``: when using ``vsbl`` as the virtual bootloader, use this
  immediately after ``virtio-blk`` to specify it as a bootable
  device and the bootable image location.
```

Comment (binbinwu1) [reply doc/developer-guides/hld/virtio-blk.rst]: thanks for advice.

**Comment by gvancuts** [comment]
I thought ACRN used `vSBL` by default to boot Linux-as-a-Guest, but we do not use this option in the launch script on the NUCs. What am I missing?

**Comment by binbinwu1** [comment]
> I thought ACRN used `vSBL` by default to boot Linux-as-a-Guest, but we do not use this option in the launch script on the NUCs. What am I missing?

It is not using vSBL to boot Linux guest on NUCs. vSBL is dedicated for APL-MRB.
For open-source project, OVMF will take the role as the virtual bootloader eventually.

**Pull request merged**